### ISSUE TITLE
Update Options

### ISSUE THREAD

**Issue by paramov** [norkunas/youtube-dl-php#153]
Hi,

Can you add new options? 
# OPTIONS
    --list-extractors                    List all supported extractors
    --extractor-descriptions             Output descriptions of all supported
                                         extractors
    --force-generic-extractor            Force extraction to use the generic
                                         extractor
Thank you

**Comment by norkunas** [comment]
Hi :)

For the `--list-extractors` you can directly call `YoutubeDl::getExtractorsList` method.
For `--extractor-descriptions` if you'll need you could just extend the `YoutubeDl` class and parse by yourself them or contribute and make a pull request :) but it's hard to parse the output which is non deterministic because the output doesn't have any separators..
And for the last one maybe you could also contribute :)
